index.d.ts

```diff
@@ -0,0 +1,20 @@
+interface NodeRequireFunction {
+  (moduleName: 'electron'): typeof Electron;
+}
+
+interface NodeRequire extends NodeRequireFunction {
+  resolve: RequireResolve;
+  cache: NodeRequireCache;
+  /**
+   * @deprecated
+   */
+  extensions: NodeExtensions;
+  main: NodeModule | undefined;
+}
+
+export declare var require: NodeRequire;
+
+// Taken from `RemoteMainInterface`
+export {app, autoUpdater, BrowserView, BrowserWindow, ClientRequest, clipboard, CommandLine, contentTracing, Cookies, crashReporter, Debugger, desktopCapturer, dialog, Dock, DownloadItem, globalShortcut, inAppPurchase, IncomingMessage, ipcMain, Menu, MenuItem, MessageChannelMain, MessagePortMain, nativeImage, nativeTheme, net, netLog, Notification, powerMonitor, powerSaveBlocker, protocol, screen, ServiceWorkers, session, shell, systemPreferences, TouchBar, TouchBarButton, TouchBarColorPicker, TouchBarGroup, TouchBarLabel, TouchBarOtherItemsProxy, TouchBarPopover, TouchBarScrubber, TouchBarSegmentedControl, TouchBarSlider, TouchBarSpacer, Tray, webContents, WebRequest} from 'electron';
```

Comment:
Adding this since it might not be exported anymore in the next version of Electron,